src/exec.js

```diff
@@ -21,6 +23,9 @@ function execSync(cmd, opts) {
     silent: common.config.silent
   }, opts);
 
+  var maxBuffer = parseInt(opts.maxBuffer, 10) || DEFAULT_MAXBUFFER_SIZE;
```

Comment:
I don't see it... why not? If someone decides to input `{ maxBuffer: "20000" }`, this line will make that work. Or am I missing something?